docs/src/main/sphinx/sql/show-catalogs.rst

```diff
@@ -12,5 +12,10 @@ Synopsis
 Description
 -----------
 
-List the available catalogs.
-The ``LIKE`` clause can be used to restrict the list of catalog names.
+List the available catalogs. 
+
+:ref:`Specify a pattern <like_operator>` in the optional `LIKE` clause to filter
```

Comment:
Would suggest making "`LIKE` clause" the link text, as the subject of the link

Reply:
I'm not following Joe's suggestion here and throughout. Jessie, did you figure out what he meant? Can I see a sample of the reworked sentence?

Reply:
he suggests to make the phrase "LIKE clause" the link to the other page .. I would leave it

Reply:
The destination of this link talks about the LIKE clause, not about specifying patterns in general, so my preference is to be specific about what info the reader will get when they click.

But it's a really nitpicky point in this case, so I think it's fine to leave as well. Just offering some clarity on what I meant.